src/components/Button.js

```diff
@@ -17,13 +16,18 @@ export const ButtonStyle = styled('button')`
   position: ${props => props.theme.position};
   padding: ${props => props.theme.padding};
   width: ${props => props.theme.width};
-  box-shadow: ${props => props.theme.shadow};
+  height: ${props => props.theme.height};
```

Comment:
Will this height be used for buttons, or for other things as well? Is the root of the theme a good place for it?